src/constants/homePage.ts

```diff
@@ -0,0 +1,3 @@
+const HOME_PAGE_LINK = "https://courtcanva.com/";
```

Comment:
It seems that this url only belongs to the navbar. Why create a folder outside the navbar file

Reply:
Because we were considering  traditionally setting these kind of URLs as constants would be more maintainable and were reusable for being used in other components possibly in the future. Lawrence seems to agree with this way in the first comment.

Reply:
Ok～，just change the name to index.ts